processor/transform/readme-to-mdx.ts

```diff
@@ -42,8 +42,10 @@ const readmeToMdx = (): Transform => tree => {
   visit(tree, 'image', (image, index, parent) => {
     if (!('data' in image)) return;
 
-    if ('url' in image) image.data.hProperties.src = image.url;
-    const attributes = toAttributes(image.data.hProperties, imageAttrs);
+    const attributes = toAttributes(
+      { ...image.data.hProperties, ...('url' in image && { src: image.url }) },
+      imageAttrs,
+    );
```

Comment:
Ahhh, interesting 🤔 this is a subtle refactor! So just ask to clarify: because we were actually *assigning* this `src` in to the `image` itself, it broke things…somewhere further along in one of our processors' "thread"? That makes sense with your fix!

Reply:
This line is where it's erroring, something higher up in the chain is somehow freezing this prop??!?! ~~What's annoying, is I can't seem to reproduce it local to this repo?~~

Just figured it out. `slate` uses `immer` and the emoji serializer is just copying all the slate node data into the mdast. I'll fix this in the main app.